src/Umbraco.Core/Models/ContentExtensions.cs

```diff
@@ -444,154 +434,116 @@ public static void PropertyValues(this IContent content, object value)
             }
         }
 
+        public static IContentTypeComposition GetContentType(this IContentBase contentBase)

```

Comment:
In variousp places we need the "content type" of an `IContentBase` but that interface does not expose a `IContentType` property (that would need to return an `IContentTypeComposition` and be newed in eg `IContent` to return an `IContentType`). Adding it would break compat, so it's implemented with this extension method.